spring-cloud-dataflow-module-deployers/spring-cloud-dataflow-module-deployer-local/src/main/java/org/springframework/cloud/dataflow/module/deployer/local/LocalModuleDeployer.java

```diff
@@ -52,6 +55,9 @@
 
 	private final RestTemplate restTemplate = new RestTemplate();
 
+	@Autowired
+	ModuleResolver moduleResolver;
```

Comment:
Changes to this class seem to be from the previous attempt, as nothing is being done with the resolver, am I correct?